connectors/zapier/connector.py

```diff
@@ -82,6 +82,11 @@ def hand_off_to_zapier(entity_name, message):
                     logging.info(has_attribution_sellv2)
                 else:
                     hand_off_to_zapier(entity_name, message)
+            elif 'completed_at' in message['payload'] and message['payload']['product_id'] == 'rating_v2_commerzbank':
+                if 'marketing_attribution' in message['payload']:
+                    logging.info(has_attribution_sellv2)
+                else:
+                    hand_off_to_zapier(entity_name, message)
```

Comment:
Basically we should add this right? I really did not understand what is the 'create' block is for and in which cases it will be triggered.
It seems to me like it will be triggered for `rating_v2_commerzbank` and we should say like `!= rating_v2_commerzbank` but I really cannot understand what is this block for.

Reply:
Technically it would be able to make a combined check on L.74 with `in` operator https://docs.python.org/3/reference/expressions.html#membership-test-operations

Reply:
```python
message['payload']['product_id'].lower()` in ['ratingv2', 'rating_v2_commerzbank']
```